### PR TITLE
Added recipe for netaddr

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: recipes/netaddr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+  home: http://github.com/drkjam/netaddr
+  license: BSD 3-clause
+  summary: 'Pythonic manipulation of IPv4, IPv6, CIDR, EUI and MAC network addresses'
+
+extra:
+  recipe-maintainers:
+    - mariusvniekerk
+    - drkjam
```
Comment (jakirkham) [inline recipes/netaddr/meta.yaml]: @drkjam, are you ok with being added as a maintainer?

**Comment by jakirkham** [comment]
Would like to hear back regarding the added maintainer first and address accordingly. Otherwise LGTM.

**Comment by mariusvniekerk** [comment]
Sure thing

**Comment by ocefpaf** [comment]
@mariusvniekerk you can remove the maintainers that did not answer and then re-add them in the feedstock once you have their agreement. That way we can merge this for you and get the feedstock going.

**Comment by mariusvniekerk** [comment]
Will do

**Comment by drkjam** [comment]
Hi,

Apologies for the delay in response. Yes, I'm happy to be named as a
maintainer.

Thanks,

Dave M.

**Comment by jakirkham** [comment]
Thanks @drkjam.

LGTM. Happy to merge once we resolve an unrelated staged-recipe issue ( https://github.com/conda-forge/staged-recipes/issues/900 ).

**Pull request merged**